releases/NOTE/NOTE.bs

```diff
@@ -369,36 +369,30 @@ table, th, td {
       <th>Underlying physical sensors</th>
     </tr>
   </thead>
-  <tfoot>
-    <tr>
-      <td>[=Relative Orientation Sensor=]</td>
-      <td>[=Accelerometer=], [=Gyroscope=], MUST NOT USE [=Magnetometer=]</td>
-    </tr>
-  </tfoot>
   <tbody>
     <tr>
       <td>[=Absolute Orientation Sensor=]</td>
       <td>[=Accelerometer=], [=Magnetometer=], AND (when present) [=Gyroscope=]</td>
     </tr>
-  </tbody>
-  <tbody>
     <tr>
       <td>[=Geomagnetic Orientation Sensor=]</td>
       <td>[=Accelerometer=], [=Magnetometer=], MUST NOT USE [=Gyroscope=]</td>
     </tr>
-  </tbody>
-  <tbody>
     <tr>
       <td>[=Gravity Sensor=]</td>
       <td>[=Accelerometer=], [=Gyroscope=]</td>
     </tr>
-  </tbody>
-  <tbody>
     <tr>
       <td>[=Linear Acceleration Sensor=]</td>
       <td>[=Accelerometer=], AND EITHER [=Gyroscope=] OR [=Magnetometer=]</td>
     </tr>
   </tbody>
+  <tfoot>
```

Comment:
Based on other versions of this file I believe the intention was not for this to be the last entry in the table and should be a &lt;tr> just like the rest.

Reply:
Should this row stay at the top as a regular TR in TBODY or at the end of the TBODY?

Reply:
I think it should stay at the top as a TR in only a single TBODY. The incorrect markup doesn't actually move the row to the bottom of the table as this change does.

Reply:
OK, updated. I didn't regenerate the HTML file version, since I just did this all on the web 😄